analyticsdataserver/settings/base.py

```diff
@@ -42,16 +42,31 @@
 
```

Comment:
We are ignoring these ones:

  MEDIA_STORAGE_BACKEND # not used
  STATICFILES_STORAGE   # not used
  SOCIAL_AUTH_EDX_OIDC_KEY # not used
  SOCIAL_AUTH_EDX_OIDC_SECRET # not used
  SOCIAL_AUTH_EDX_OIDC_ID_TOKEN_DECRYPTION_KEY # not used
  SOCIAL_AUTH_EDX_OIDC_URL_ROOT # not used
  SOCIAL_AUTH_EDX_OIDC_PUBLIC_URL_ROOT # not used
  SOCIAL_AUTH_REDIRECT_IS_HTTPS # not used
  SOCIAL_AUTH_EDX_OIDC_LOGOUT_URL # not used
  SOCIAL_AUTH_EDX_OIDC_ISSUER # not used
  SOCIAL_AUTH_EDX_OAUTH2_KEY # not used
  SOCIAL_AUTH_EDX_OAUTH2_SECRET # not used
  SOCIAL_AUTH_EDX_OAUTH2_ISSUER # not used
  SOCIAL_AUTH_EDX_OAUTH2_URL_ROOT # not used
  SOCIAL_AUTH_EDX_OAUTH2_LOGOUT_URL # not used
  BACKEND_SERVICE_EDX_OAUTH2_KEY # not used
  BACKEND_SERVICE_EDX_OAUTH2_SECRET # not used
  BACKEND_SERVICE_EDX_OAUTH2_PROVIDER_URL # not used
  EXTRA_APPS # not used
  EDX_DRF_EXTENSIONS # not used
  SESSION_EXPIRE_AT_BROWSER_CLOSE # not used
  API_ROOT # not used

because they are not used, is that correct?

Reply:
I have added these variables. These variables are overriding in meta/main.yml.